doc/COMPILE.md

```diff
@@ -82,17 +82,22 @@ apt-get install pdflatex \
         texlive-math-extra
 ```
 
-For the plugins, please refer to the README.md of the respective plugin.
-For example, for CentOS:
+For the plugins, please refer to the [README.md](https://www.libelektra.org/plugins/readme) of the respective plugin.
 
+A small subset of plugins to get you started:
+for CentOS:
```

Comment:
This looks a bit odd when rendered, as there is no newline.

Maybe use a list to indent "for CentOS" and other sections (or use some other structure).
e.g.:
```suggestion
A small subset of plugins to get you started:
* for CentOS:
```